### PR TITLE
Added Array, logger, devel, subsystem, telemetry commands in Kouros m…

### PR DESCRIPTION
**Pull request by vishal-shakya-ssir**
Kindly review the changes of added Array, logger, devel, subsystem, telemetry commands in Kouros module

### CODE REVIEW

File: src/kouros/pos/grpc_manager.go [hunk 1]
```diff
@@ -73,8 +73,209 @@ func (p *POSGRPCManager) ListDevices() (*pb.ListDeviceResponse, error) {
...
-func (p *POSGRPCManager) CreateArray(param pb.CreateArrayRequest_Param) (*pb.CreateArrayResponse, error) {
+func (p *POSGRPCManager) CreateArray(param *pb.CreateArrayRequest_Param) (*pb.CreateArrayResponse, error) {
 	command := "CREATEARRAY"
-	req := &pb.CreateArrayRequest{Command: command, Rid: utils.GenerateUUID(), Requestor: "cli", Param: &createArrayPB}
+	req := &pb.CreateArrayRequest{Command: command, Rid: utils.GenerateUUID(), Requestor: p.requestor, Param: param}
 	return grpc.SendCreateArray(p.connection, req)
 }
+
+// Add device command add spare device in PoseidonOS Array
+// The function takes a byte array in JSON format as parameter and returns response in JSON format
```
Comment (aswinssir) [inline src/kouros/pos/grpc_manager.go:83]: Please correct the description. Now the Parameters are not in JSON format

File: src/kouros/pos/pos_manager.go [hunk 2]
```diff
@@ -23,10 +23,102 @@ type POSManager interface {
...
+	ListDevices() (*pb.ListDeviceResponse, error)
 
 	// Create Array method creates an array in PoseidonOS
 	// The function takes a command in JSON format as the parameter
 	// A successful Create call returns response in JSON format
-	CreateArray(param pb.CreateArrayRequest_Param) (response *p.CreateArrayResponse, err error)
+	CreateArray(param *pb.CreateArrayRequest_Param) (*pb.CreateArrayResponse, error)
+
+	// Add device command add spare device in PoseidonOS Array
+	// The function takes a byte array in JSON format as parameter and returns response in JSON format
```
Comment (aswinssir) [inline src/kouros/pos/pos_manager.go:34]: Please change the comment related to JSON format.

File: src/kouros/pos/grpc_manager.go (hunk 1 above)
Comment (vishal-shakya-ssir) [reply src/kouros/pos/grpc_manager.go:83]: oh Thanks for pointing out, I will correct the description

File: src/kouros/pos/pos_manager.go (hunk 2 above)
Comment (vishal-shakya-ssir) [reply src/kouros/pos/pos_manager.go:34]: I will correct the description

**Review by mjlee34 — APPROVED** [review]
LGTM. Thanks for the quick development.

**Review by aswinssir — APPROVED** [review]

**Pull request merged**